main/controller/team_controller.py

```diff
@@ -0,0 +1,29 @@
+from django.shortcuts import render, redirect
+
+from main.service import team_service
+from main.model_form.team_form import TeamForm
+from django.views import View
+
+
+class CreateTeamView(View):
+    team_form = TeamForm
+
+    def get(self, request):
```

Comment:
I understand the confusion here. Just think of it this way. Say we make the following request:
GET /team/create
 In the current implementation, it will result in a form for user to enter team details

There are two problems:
1. As per REST principle, 
GET /team/<:id> 
should be used to fetch a team instance by id.
Now "create" string will be taken as "id" as it follows "/team". 

As mentioned earlier,
GET /team - gets representation of all teams
POST /team - create a single team instance
PUT /team/<:id> - replaces existing team object with given id by new team object provided in the payload
GET /team/<:id> - gets a team by id

As you can see, /team in one way or other should always interact with "Team" table

Now we want one path where we can simply get a html form which when submitted will POST to "/team" all the form data. This html form has nothing to do with "Team" table.

So for any such paths which do not deal with tables, create another route like "/form/team" by which the server knows this url has nothing do with Team table but is sinply a page with normal html contents.